frontends/election-manager/src/app.test.tsx

```diff
@@ -389,6 +391,33 @@ test('L&A (logic and accuracy) flow', async () => {
     expect.stringContaining('Hand-marked paper ballot test deck')
   );
   expect(container).toMatchSnapshot();
+
+  // Test printing full test deck tally
+  userEvent.click(screen.getByText('L&A'));
+  userEvent.click(screen.getByText('Print Full Test Deck Tally Report'));
+  await screen.findByText('Printing');
+  expect(printer.print).toHaveBeenCalledTimes(5);
+  expect(mockKiosk.log).toHaveBeenCalledWith(
+    expect.stringContaining(LogEventId.TestDeckTallyReportPrinted)
+  );
+  const fullTestDeckTallyReport = document.querySelector(
+    '#full-test-deck-tally-report'
+  ) as HTMLElement;
+  const expectedTallies: { [tally: string]: number } = {
+    '104': 6,
+    '52': 12,
+    '36': 2,
+    '24': 2,
+    '12': 3,
+    '8': 1,
+    '4': 1,
+    '0': 10,
+  };
+  for (const [tally, times] of Object.entries(expectedTallies)) {
+    expect(domGetAllByText(fullTestDeckTallyReport, tally).length).toEqual(
```

Comment:
Extending the approach from testing the contents of the zero report, this seemed preferable to a snapshot as we are trying to minimize snapshots.